src/posix/platform/trel.cpp

```diff
@@ -52,7 +52,6 @@
 #if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
 
 #define TREL_MAX_PACKET_SIZE 1400
```

Comment:
Shall we also make a configurable macro for this? Or we can just make it a constant variable instead of a macro.

Reply:
What's tradeoff between making it a constant variable and a macro?

Reply:
`const` pros over macro: 
- Doesn't involve preprocessor. Less error-prone.
- More concise and readable. 

macro pros over `const`:
- Configurable in headers and build options.

Reply:
Ok, changed it to the const static variable.